net/dns/manager.go

```diff
@@ -46,10 +43,11 @@ var (
 // the request will be dropped to avoid blocking the caller.
 func maxActiveQueries() int32 {
 	if runtime.GOOS == "ios" {
-		// For memory paranoia reasons on iOS, match the
-		// historical Tailscale 1.x..1.8 behavior for now
-		// (just before the 1.10 release).
-		return 64
+		// For memory paranoia reasons, we have a smaller limit on iOS.
```

Comment:
No need to have the history of changes in the comment, git history will do that.

96 seems fine, although I'd be tempted to remove the special case entirely and see what happens.

Reply:
> Given I've seen this limit hit a few times, it seems prudent to boop it slightly to 96.

Since that limit of 64 was added, we dropped support for iOS versions older than iOS 15. We now have 50 MBytes available on all iOS devices instead of 15 MBytes (https://tailscale.com/blog/go-linker/)

I think we can drop special cases about iOS memory at this point.

Reply:
I'll remove the special-case.